configs/.env.production

```diff
@@ -0,0 +1,4 @@
+REACT_APP_API_URL='https://dev-api.clearlydefined.io'
```

Comment:
Should this not be the actual production url?

Reply:
As mentioned in the comment below (that was actually in the api/clearlyDefined.js file) "this needs to be replaced when we do a prod deployment. We want a "production" build deployed in
the dev environment but of course it will need to point to the dev server"

Reply:
Default should be the production url, given this file is production-specific. 
The previous comment should indeed be superseded by what we are doing. Hence I would recommend to change REACT_APP_API_URL with the production one for what's concerning .env.production.